src/Spryk/Model/Spryk/Executor/SprykExecutor.php

```diff
@@ -77,9 +77,9 @@ class SprykExecutor implements SprykExecutorInterface
     protected ConditionMatcherInterface $conditionMatcher;
 
     /**
-     * @var array<\SprykerSdk\Spryk\Model\Spryk\Definition\SprykDefinitionInterface>
+     * @var array<string, \SprykerSdk\Spryk\Model\Spryk\Executor\ConditionMatcher\ConditionMatcherInterface>
```

Comment:
The wrong interface is used here, it MUST be `SprykDefinitionInterface`

Reply:
Already fixed in later commit